atomic_reactor/plugins/add_labels_in_df.py

```diff
@@ -133,8 +133,8 @@ def __init__(self, workflow, labels=None, dont_overwrite=None,
             raise RuntimeError("equal_labels have to be list")
 
     def generate_auto_labels(self, platform: str) -> Dict[str, str]:
-        # build date
-        dt = datetime.datetime.utcfromtimestamp(atomic_reactor_start_time)
+        # build date, using the current time here to be > start time of koji build
+        dt = datetime.datetime.utcfromtimestamp(time.time())
```

Comment:
Why not use the pipeline run start time as the build date?

Reply:
changed it to use start time